docs_src/microservices/application/Triggers.md

```diff
@@ -1,4 +1,4 @@
-Triggers determine how the app functions pipeline begins execution. In the simple example provided above, an HTTP trigger is used. The trigger is determine by the `configuration.toml` file located in the `/res` directory under a section called `[Binding]`. Check out the [Configuration Section](#configuration) for more information about the toml file.
+Triggers determine how the app functions pipeline begins execution. The trigger is determine by the `configuration.toml` file located in the `/res` directory under a section called `[Binding]`. Check out the [Configuration Section](#configuration) for more information about the toml file.
```

Comment:
determined

Reply:
Configuration Section doesn't look to be rendering as a link

Reply:
> Configuration Section doesn't look to be rendering as a link

This is because I haven't created it yet... ;-) Planning to put that in a separate PR.